lms/static/js/learner_dashboard/spec/course_card_view_spec.js

```diff
@@ -13,8 +13,13 @@ describe('Course Card View', () => {
     const setupView = (data, isEnrolled, collectionCourseStatus) => {
         const programData = $.extend({}, data);
         const context = {
-            courseData: {},
+            programData,
```

Comment:
uneducated question: why was there a switch from `courseData` to `programData` here? If this is for a course card... should we be making this change?

Reply:
nevermind, I see we're adding programData at the top but courseData is still included below